integration/spark/shared/src/test/java/io/openlineage/spark/agent/util/PathUtilsTest.java

```diff
@@ -196,6 +196,64 @@ void testFromCatalogTableWithGlue() throws URISyntaxException {
+  void testFromCatalogTableWithEMRGlue() throws URISyntaxException {
+    hadoopConf.set(
+        "hive.metastore.client.factory.class",
+        "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory");
+    sparkConf.set("spark.sql.catalogImplementation", "hive");
+    sparkConf.set("hive.metastore.glue.catalogid", "123456789012");
+
+    when(catalogTable.provider()).thenReturn(Option.apply("hive"));
+    when(catalogTable.storage()).thenReturn(catalogStorageFormat);
+    TableIdentifier tableIdentifier = mock(TableIdentifier.class);
+    when(catalogTable.identifier()).thenReturn(tableIdentifier);
+    when(tableIdentifier.database()).thenReturn(Option.apply("database"));
+    when(tableIdentifier.table()).thenReturn("mytable");
+    when(catalogStorageFormat.locationUri())
+        .thenReturn(Option.apply(new URI("s3://bucket/warehouse/mytable")));
+
+    DatasetIdentifier datasetIdentifier = PathUtils.fromCatalogTable(catalogTable, sparkSession);
+    assertThat(datasetIdentifier)
+        .hasFieldOrPropertyWithValue("name", "warehouse/mytable")
+        .hasFieldOrPropertyWithValue("namespace", "s3://bucket");
+    assertThat(datasetIdentifier.getSymlinks()).hasSize(1);
+    assertThat(datasetIdentifier.getSymlinks().get(0))
+        .hasFieldOrPropertyWithValue("name", "table/database/mytable")
+        .hasFieldOrPropertyWithValue("namespace", "arn:aws:glue:us-east-1:123456789012")
+        .hasFieldOrPropertyWithValue("type", SymlinkType.TABLE);
+  }
+
+  @Test
+  @SetEnvironmentVariable(key = "AWS_DEFAULT_REGION", value = "us-east-1")
+  void testFromCatalogTableWithAthenaGlue() throws URISyntaxException {
+    sparkConf.set(
+        "hive.metastore.client.factory.class",
+        "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory");
+    sparkConf.set("spark.sql.catalogImplementation", "hive");
+    hadoopConf.set("hive.metastore.glue.catalogid", "123456789012");
+
+    when(catalogTable.provider()).thenReturn(Option.apply("hive"));
+    when(catalogTable.storage()).thenReturn(catalogStorageFormat);
+    TableIdentifier tableIdentifier = mock(TableIdentifier.class);
+    when(catalogTable.identifier()).thenReturn(tableIdentifier);
+    when(tableIdentifier.database()).thenReturn(Option.apply("database"));
+    when(tableIdentifier.table()).thenReturn("mytable");
+    when(catalogStorageFormat.locationUri())
+        .thenReturn(Option.apply(new URI("s3://bucket/warehouse/mytable")));
+
+    DatasetIdentifier datasetIdentifier = PathUtils.fromCatalogTable(catalogTable, sparkSession);
+    assertThat(datasetIdentifier)
+        .hasFieldOrPropertyWithValue("name", "warehouse/mytable")
+        .hasFieldOrPropertyWithValue("namespace", "s3://bucket");
+    assertThat(datasetIdentifier.getSymlinks()).hasSize(1);
+    assertThat(datasetIdentifier.getSymlinks().get(0))
+        .hasFieldOrPropertyWithValue("name", "table/database/mytable")
```

Comment:
Any reason for having name parts `table/database/mytable` slash separated?
In other cases OpenLineage follows dot separator for table names.

Reply:
https://github.com/OpenLineage/OpenLineage/issues/2765#issuecomment-2165579162